### PR TITLE
Fix FingerprintJS regex

### PR DESCRIPTION
**Pull request by VictorLeP**
Apply correct regex per https://github.com/AliasIO/wappalyzer/pull/4053#issuecomment-868276095

### CODE REVIEW

**Comment by nurbek91** [comment]
examples:
https://www.drtusz.pl/
https://people.cs.kuleuven.be/~victor.lepochat/mwe.html
https://uk.wallapop.com/
https://resolume.com/
https://www.as-web.jp/
https://www.comuniazo.com/

**Review by VictorLeP — COMMENTED** [review]
Confirmed to work [here](https://github.com/AliasIO/wappalyzer/pull/4053#issuecomment-869530798), so this can be merged,

**Comment by VictorLeP** [comment]
Could this PR still be merged if the changes are OK? It would be great to have this included for the July HTTP Archive crawl.

File: src/technologies.json [hunk 1]
```diff
@@ -7155,7 +7155,7 @@
       "saas": true,
       "scripts": [
         "fingerprint(?:/fp)?(\\d)?(?:\\.min)?\\.js\\;version:\\1",
-        "fingerprintjs(?:\\-pro|2)?(?:@|/)([\\d.]+)(?:\\-)?([\\d.a-z]+)?\\;version:\\1\\2"
+        "fingerprintjs(?:\\-pro|2)?(?:@|/)(\\d.*?)?/\\;version:\\1"
```
[the remark this answers is not in the document]
Comment (nurbek91) [reply src/technologies.json:7158]: Hey  , yes confirm .

**Comment by AliasIO** [comment]
Done, thanks!

**Pull request merged**